cli/azd/CHANGELOG.md

```diff
@@ -1,15 +1,35 @@
 # Release History
 
-## 1.18.0-beta.1 (Unreleased)
+## 1.18.0
 
 ### Features Added
 
-### Breaking Changes
+- [[5380]](https://github.com/Azure/azure-dev/pull/5380) Fix project name validation to prevent service packaging failures with invalid characters.
```

Comment:
```suggestion
- [[5380]](https://github.com/Azure/azure-dev/pull/5380) Improve project name validation to prevent service packaging failures with invalid characters.
```